build/project-template/internal/nsld.sh

```diff
@@ -0,0 +1,53 @@
+ #/bin/bash
+
+ set +o posix
```

Comment:
Maybe we should add a comment here. E.g. something like `Ignore errors while inserting environment variables (some could be readonly)`